src/components/Request/Pay/Views/Initial.view.tsx

```diff
@@ -14,6 +14,8 @@ import TokenSelector from '@/components/Global/TokenSelector/TokenSelector'
 import { switchNetwork as switchNetworkUtil } from '@/utils/general.utils'
 import { ADDRESS_ZERO, EPeanutLinkType, RequestStatus } from '../utils'
 
+const ERR_NO_ROUTE = 'No route found to pay in this chain and token'
```

Comment:
nit: maybe better to standardize these somewhere?

Reply:
sounds good